src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { nowUOrange } from '../../assets';
+import {
+    topNavLinks,
+    bottomNavLinks,
+    mobileNavLinks,
+} from '../../utils/constants';
+import NavLinkSection from './navLinkSection/NavLinkSection';
+import NavDropdown from './navDropdown/NavDropdown';
+import classes from './Navbar.module.scss';
+
+// import NewsletterSignupModal from '../newsletter-signup-modal/NewsletterSignupModal';
+
+const Navbar = ({ newsModal }) => {
+    const [dropdownNavDisplay, setDropdownNavDisplay] = useState(false);
```

Comment:
`displayNavDropdown` as it's a boolean which controls whether something is displayed and the object being displayed is the `NavDropdown` not `DropdownNav`

Reply:
I believe you are talking about a naming convention?

Reply:
If so, I changed them to displayNavDropdown which is a good idea because the links have a display key also.

Reply:
oh nvm, I see it.